tutorials-and-examples/inference-servers/jetstream/maxtext/single-host-inference/README.md

```diff
@@ -121,9 +121,11 @@ Completed unscanning checkpoint to gs://BUCKET_NAME/final/unscanned/gemma_7b-it/
 
 ## Deploy Maxengine Server and HTTP Server
```

Comment:
High level: why put the terraform files into the "tutorials and examples" folder rather than the benchmarks folder? The benchmarks folder is meant to have the terraform automation for the workloads, and it seems like we could organize this to have the "tutorials and examples" to refer to the terraform scripts in the benchmarking suite as needed.

Reply:
discussed over chat, purpose is to have one off tutorial, not to add to benchmarking suite. Thus choices here relate more to tutorials-and-examples/ work. Refactoring tf files to happen in follow up PR.